File: backend/src/models/facture.js
class Facture{
    constructor(id, id_promotion, prix){
        this._id = id
        this._id_promotion = id_promotion
        this._prix = prix
    }
    /* ######################################### GETTER #########################################*/
    get id(){
        return this._id
    }
    get id_promotion(){
        return this._id_promotion
    }
    get prix()
    {
        return this._prix
    }

    /* ######################################### SETTER #########################################*/

    set id(id){
        this._id = id
    }
    set id_promotion(id_promotion){
        this._id_promotion = id_promotion
    }
    set prix(prix){
        this._prix = prix
    }
}

module.exports = Facture

File: android/PetSitterISI/app/src/main/java/com/example/petsitterisi/entitees/Facture.java
package com.example.petsitterisi.entitees;

public class Facture {

    private int  idFacture;
    private int  idPromotion;
    private Double  prix;

    public Facture(int idFacture, int idPromotion, Double prix) {
        this.idFacture = idFacture;
        this.idPromotion = idPromotion;
        this.prix = prix;
    }

    public int getIdFacture() {
        return idFacture;
    }

    public int getIdPromotion() {
        return idPromotion;
    }

    public Double getPrix() {
        return prix;
    }
}
